### DISCUSSION TITLE
Fragen und Antworten

### DISCUSSION THREAD

**Discussion by stweil** [UB-Mannheim/escriptorium#10] [Q&A]
Bitte schreiben Sie einen neuen Beitrag in der Kategorie _Q & A_, wenn Sie eine Frage haben, die auch für andere Nutzer*innen relevant sein könnte. Das eScriptorium-Team der UB Mannheim wird dann direkt hier antworten.

Das Team hat auch eine Frage an Sie:

**Wie zufrieden sind Sie mit unserer Mannheimer eScriptorium-Instanz?**

Gerne hier antworten!

Weitere Fragen (und hoffentlich auch Antworten) stehen hier:
https://github.com/UB-Mannheim/escriptorium/discussions/categories/q-a

Und wenn Sie selbst eine Frage haben, die dort noch nicht zu finden ist, verwenden Sie diesen Link:
https://github.com/UB-Mannheim/escriptorium/discussions/new?category=q-a

**Comment by lawi21** [comment]


**Reply by lawi21** [reply]
Dass Sie die URL http://localhost:8000/ aufrufen können, setzt voraus, dass der Server läuft. Hierfür muss der Befehl `python manage.py runserver --settings escriptorium.local_settings" erfolgreich ausgeführt worden sein. Wiederholen Sie den Schritt einfach nochmals.
